app/views/taxons/edit.html.erb

```diff
@@ -10,4 +10,9 @@
       input_html: { class: :select2, multiple: true, include_blank: true } %>
 
   <%= f.submit I18n.t('views.taxons.edit_button'), class: "btn btn-lg btn-success" %>
+
+  <%= link_to 'Delete taxon', taxon_path(taxon.content_id),
```

Comment:
Can we move this to the header perhaps? Having it next to the submit button makes me nervous.